tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -1467,6 +1467,153 @@ public function test_intent_error_throws_exception() {
+	/**
+	 * Test that a customer-cancelled redirect (e.g. Klarna popup closed) during
+	 * process_upe_redirect_payment does NOT fail the order and redirects to checkout.
+	 */
+	public function test_process_upe_redirect_payment_cancellation_does_not_fail_order() {
+		$payment_intent_id = 'pi_mock';
+		$order             = WC_Helper_Order::create_order();
+		$order_id          = $order->get_id();
+
+		list( $amount ) = $this->get_order_details( $order );
+
+		$payment_intent_mock                       = self::MOCK_CARD_PAYMENT_INTENT_TEMPLATE;
```

PHPMD (2.15.0)

[error] The variable $payment_intent_mock is not named in camelCase. (undefined)

(CamelCaseVariableName)